src/library/database/database_config.py

```diff
@@ -25,12 +23,12 @@ def assemble_db_connection(cls, v: Optional[str], info: ValidationInfo) -> str:
         return f'postgresql+asyncpg://{user}:{password.get_secret_value()}@{server}/{db}'
 
     model_config = SettingsConfigDict(
-        env_file='.env',
+        env_file='../../.env',
```

Comment:
bad practice to use relative path, probably you want to use something like: `.env`

Reply:
done